modules/ROOT/pages/site-structure.adoc

```diff
@@ -0,0 +1,57 @@
++
+Each search result links to the appropriate page based on the type of the element, such as a content article page, an API details page, or others.
+* News list page: Contains a list of the different content articles, such as news or announcements (*CMS Collection* component).
+* News detail page: Displays the full content of a specific content article (*CMS Single Item* component).
+* APIs and API groups catalog page: Contains the full catalog of APIs and API groups available in your community, for consumers to explore and discover (xref:acm-lightning-components.adoc#api-catalog[API Catalog] component).
+* API and API groups details page: Displays the documentation and technical content of an API or API group, including the API console and a way to request access to the API.
++
+The content of this page is tailored to the type of object being accessed using page variations. You can find more information in the Salesforce Community Cloud documentation page https://help.salesforce.com/articleView?id=community_builder_manage_pages_overview.htm&type=5[Manage Your Community's Pages and Their Properties in Experience Builder, role=external, window=_blank].
++
+** Individual API page variation: This default page variation is used when an API object is opened from the *API Card*, *API Carousel*, or *API Catalog* components.
++
+The API page variation includes the API name, icon, and description (xref:acm-lightning-components.adoc#api-header[API Header] component), its markdown documentation from Exchange (xref:acm-lightning-components.adoc#api-documentation-viewer[API Documentation Viewer] component), the API spec (xref:acm-lightning-components.adoc#api-console[API Console] component), links to download the API spec (xref:acm-lightning-components.adoc#spec-downloader[Spec Downloader] component), and the button to request access (xref:acm-lightning-components.adoc#api-access-requester[API Access Requester] component).
+** API group page variation: This page variation displays an API group when an API group object is opened from the *API Card*, *API Carousel*, or *API Catalog* components.
++
+The API group page variation includes the API group name, icon, and description (xref:acm-lightning-components.adoc#api-header[API Header] component), its markdown documentation from Exchange (xref:acm-lightning-components.adoc#api-documentation-viewer[API Documentation Viewer] component), an API carousel displaying the APIs that are part of this API group (xref:acm-lightning-components.adoc#api-carousel[API Carousel] component, with the *Show* setting configured for *Children*), and the button to request access to the API group (xref:acm-lightning-components.adoc#api-access-requester[API Access Requester] component).
++
+To configure this variation, assign a new audience to this page variation for the API group record type:
```

Comment:
This needs to be a Task topic, with its own title, prereqs, steps, next steps, and See Also subtopics:
```suggestion
Configure this variation by following the steps in Configure API Page Variations.
```
And then create that task topic using that title (or some other you decide upon), a brief intro to the purpose of the task, and the following steps:

Reply:
Updated to move the configuration information out of the bullet list.